xdsl/dialects/llvm.py

```diff
@@ -185,6 +184,63 @@ def from_size_and_type(size: int | IntAttr, type: Attribute):
         return LLVMArrayType([size, type])
 
 
+@irdl_attr_definition
+class LLVMVoidType(ParametrizedAttribute, TypeAttribute):
+    name = "llvm.void"
+
+    def print_parameters(self, printer: Printer) -> None:
+        ...
+
+    @staticmethod
+    def parse_parameters(parser: Parser) -> list[Attribute]:
+        return []
```

Comment:
What happens if you don't specify it? It should still work, right?

Reply:
Ah no idea, I thought it was required, let me check